docs/reference/module-options.md

```diff
@@ -136,4 +138,257 @@ You might use that value as a fallback for user-editable fields.
+
+#### Example
+
+```javascript
+// modules/feature/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    label: 'Featured Article'
+  },
+  // ...
+}
+```
+
+### `localized`
+
+Defaults to `true`. If set to `false`, this doc type will _not_ be included in the locale system. This means there will be only one version of each doc, regardless of whether multiple locales (e.g., for languages or regions) are active. The "users" piece disables localization in this way.
+
+#### Example
+
+```javascript
+// modules/administrative-category/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    localized: false
+  },
+  // ...
+}
+```
+
+### `sort`
+
+The `sort` option for a doc type defines a sorting order for requests to the database for that type. This setting generally follows the [MongoDB `$sort` aggregation](https://docs.mongodb.com/manual/reference/operator/aggregation/sort/) syntax. The option is set to an object containing field name keys with `1` as a property value for ascending order and `-1` for descending order.
+
+The default sort for all doc types is `{ updatedAt: -1 }`, meaning it returns documents based on the `updatedAt` property (the date and time of the last update) in descending order. The `sort` object can have multiple keys for more specific sorting.
+
+#### Example
+
+This `sort` setting will return articles first based on a custom `priority` field in ascending order, then by the core `updatedAt` property in descending order.
+
+```javascript
+// modules/article/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    sort: {
+      priority: 1,
+      updatedAt: -1
+    }
+  },
+  fields: {
+    add: {
+      priority: {
+        type: 'integer',
+        min: 1,
+        max: 5
+      },
+      // ...
+    }
+  }
+  // ...
+}
+```
+
+### `slugPrefix`
+
+Set `slugPrefix` to a string to prepend all [slugs](glossary.md#slug) for docs of this type. This can be useful to help prevent slugs, which must be unique for each doc in the database, from being reserved in some cases. For example, Apostrophe image docs have the `slugPrefix` value of `'image-'` so images, which do not typically have public pages, do not accidentally reserve a more reader-friendly slug.
+
+#### Example
+
+```javascript
+// modules/article-category/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    slugPrefix: 'category-'
+  },
+  // ...
+}
+```
+
+## Options for all piece modules
+
+Option settings in this section apply to all piece modules (those that extend `@apostrophecms/piece-type`).
+
+<!-- - [`createControls`](#createcontrols)
+- [`editControls`](#editcontrols) -->
+- [`pluralLabel`](#plurallabel)
+- [`perPage`](#perpage)
+- [`publicApiProjection`](#publicapiprojection)
+- [`quickCreate`](#quickcreate)
+- [`searchable`](#searchable)
+
+<!-- ### `createControls`
```

Comment:
Don't document these, right? They're pretty serious UI editing options.

Reply:
createControls and editControls are dead code.